### ISSUE TITLE
Issue with Styling on freshly created Fullstack App

### ISSUE THREAD

**Issue by codeitlikemiley** [rust-lang/rust#118008]
I installed dioxus cli (fresh) , create a new fullstack app , start the app

```rust
dx build --features web --release
cargo run --features ssr --release
```

I expected to see this happen: 
<img width="1616" alt="Screenshot 2023-11-17 at 7 31 27 PM" src="https://github.com/rust-lang/rust/assets/28816690/2e76f02e-0541-4537-af9a-d4a5d7377ffc">

Instead, this happened: 
<img width="1616" alt="Screenshot 2023-11-17 at 7 22 24 PM" src="https://github.com/rust-lang/rust/assets/28816690/d7cd899f-3796-48fb-88af-2f75ac04e8fe">

No Styling even for a simple button, i only manage to see the expected after i navigate to /blog/{id} then press cmd + r or f5 (refresh)

**Comment by saethlin** [comment]
This doesn't look like a Rust bug. Have you reported this to dioxus?

**Comment by Nilstrieb** [comment]
You should report this to dioxus, this is the wrong place. https://github.com/DioxusLabs/dioxus not sure whether that is the right place but they will tell you what to do if it isn't.
